examples/tidal_barrage/modules/input_barrages.py

```diff
@@ -0,0 +1,55 @@
+from modules.parameterisations import *
```

Comment:
Could you add a docstring at the top very briefly, even if just one line, describing what goes into each module?

Reply:
amended accordingly, added short docstrings to be more specific